### PR TITLE
Remove "the button" wording

### PR DESCRIPTION
**Pull request by maximiliankolb**
* Add "In the {ProjectWebUI} where missing
* Drop various unnecessary whitespace
* Remove "at the bottom" because "Submit" is always at the end
* Remove emphasis for ">"
* Split sentence after "navigate to X > Y"
* Use "{endash}" in template names where applicable

I was unsure which wording we use, so I grep'ed for it. Looks like "Click *X*" is sufficient. I made this PR because it's also more minimalistic and due to the convention, users can always click on **bold** words.

I left in some instances that I was unsure how to reword, esp. when we describe something that is next to a button etc.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)

### CODE REVIEW

File: guides/common/modules/proc_assigning-a-puppet-class-to-an-individual-host.adoc [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Click on the *Edit* button of the host you want to add the `ntp` Puppet class to.
+. Click *Edit* of the host you want to add the `ntp` Puppet class to.
```
Comment (asteflova) [inline guides/common/modules/proc_assigning-a-puppet-class-to-an-individual-host.adoc:6]:
```suggestion
. Click *Edit* for the host you want to add the `ntp` Puppet class to.
```

Comment (asteflova) [reply guides/common/modules/proc_assigning-a-puppet-class-to-an-individual-host.adoc:6]:
```suggestion
. Locate the host you want to add the `ntp` Puppet class to  and click *Edit*.
```

Comment (asteflova) [reply guides/common/modules/proc_assigning-a-puppet-class-to-an-individual-host.adoc:6]: Neither seems great. Can you think of something better?

File: guides/common/modules/proc_customizing-an-ansible-playbook-to-reconfigure-hosts.adoc [hunk 2]
```diff
@@ -5,7 +5,7 @@ You can customize and rerun the Ansible playbook to reconfigure existing hosts.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Applications > App Instances*.
-. Select the application instance you want to deploy and click the *Run Playbook - customize first* button from the drop down menu.
+. Select the application instance you want to deploy and click *Run Playbook {endash} customize first* from the drop down menu.
```
Comment (asteflova) [inline guides/common/modules/proc_customizing-an-ansible-playbook-to-reconfigure-hosts.adoc:8]:
```suggestion
. Select the application instance you want to deploy and select *Run Playbook {endash} customize first* from the drop-down menu.
```
drop-down menu with a hyphen per the IBM guide

"Select" seem like the term to use for drop downs, but I'm not sure to be honest.

**Review by asteflova — APPROVED** [review]

File: guides/common/modules/proc_assigning-a-puppet-class-to-an-individual-host.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_assigning-a-puppet-class-to-an-individual-host.adoc]: I like the second suggestion and applied it.

File: guides/common/modules/proc_customizing-an-ansible-playbook-to-reconfigure-hosts.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_customizing-an-ansible-playbook-to-reconfigure-hosts.adoc]: Applied both suggestions.

**Review by maximiliankolb — COMMENTED** [review]
Applied both suggestions @asteflova Does that work for you?

File: guides/common/modules/proc_creating-an-application-instance.adoc [hunk 3]
```diff
@@ -4,7 +4,8 @@
 You can create xref:Application_Instances_{context}[] as a blueprint for users to provision and configure hosts.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Application > App Instances* and click the *New Application Instance* button.
+. In the {ProjectWebUI}, navigate to *Application > App Instances*.
```
Comment (asteflova) [inline guides/common/modules/proc_creating-an-application-instance.adoc:7]:
```suggestion
. In the {ProjectWebUI}, navigate to *Application* > *App Instances*.
```

**Review by asteflova — APPROVED** [review]
Coffee helped me find one more markup issue. Other than that, I don't see anything else. I like this PR, it was a great idea to fix these :)

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   42ee0d15d..9c7e5d6b2  3.9 -> 3.9
   f8171e339..f64e3a08e  3.8 -> 3.8
   39092db4f..57fb520cb  3.7 -> 3.7

**Pull request merged**